### PR TITLE
Reference std.digest.fromHexString in doc of std.conv.hexString

### PR DESCRIPTION
**Pull request by 0xEAB**
Documents that add similar function for run time use cases has been added and warns about their differences.
(I'm not a native English speaker. Please proof-read.)

### CODE REVIEW

File: std/conv.d [hunk 1]
```diff
@@ -5611,6 +5611,16 @@ Params:
 
 Returns:
     a `string`, a `wstring` or a `dstring`, according to the type of hexData.
+
+See_Also:
+    While this template performs the conversion during compile time,
+    the function $(REF fromHexString, std, digest) can be used for run time
+    conversions. However, these functions are not drop-in replacements and
```
Comment (thewilsonator) [inline std/conv.d:5618]:
```
Use $(REF fromHexString, std, digest) for run time conversions. Note, these functions are not drop-in replacements and have different input requirements. This template uses builtin $(LINK2 $(ROOT_DIR)spec/lex.html#hex_string, hex strings), see  $(REF fromHexString, std, digest) for its requirements.
```

Comment (0xEAB) [reply std/conv.d:5618]: > This template uses builtin hex strings.

No, it doesn’t. But it shares their syntax.

Comment (0xEAB) [reply std/conv.d:5618]: Nevertheless, thanks for the suggestion. I’ll incorporate it in my next revision.

Comment (0xEAB) [reply std/conv.d]: Please let me know what you think of my changes.

**Review by thewilsonator — APPROVED** [review]

**Pull request merged**